Library/Homebrew/requirements/language_module_requirement.rb

```diff
@@ -12,11 +12,22 @@ def initialize(language, module_name, import_name = nil)
 
   satisfy(:build_env => false) { quiet_system(*the_test) }
 
-  def message; <<-EOS.undent
-    Unsatisfied dependency: #{@module_name}
-    Homebrew does not provide #{@language.to_s.capitalize} dependencies; install with:
-      #{command_line} #{@module_name}
+  def message
+    s = <<-EOS.undent
+      Unsatisfied dependency: #{@module_name}
+      Homebrew does not provide special #{@language.to_s.capitalize} dependencies; install with:
+        `#{command_line} #{@module_name}`
     EOS
+
+    unless [:python, :perl, :ruby].include? @language
```

Comment:
^ 👍 Yup. We essentially want to make sure if a formula says:

```ruby
depends_on "virtualenv" => :python
```
Users aren't told they may need to `brew install python`.